src/nyc_trees/gulpfile.js

```diff
@@ -199,10 +200,19 @@ function browserifyTask(bundler) {
 }
 
 gulp.task('sass', ['clean'], function() {
-    return gulp.src('sass/main.scss')
-        .pipe(sourcemaps.init())
-        .pipe(sass({style: 'compressed'}))
-        .pipe(sourcemaps.write('./'))
+    var autoprefixProcessor = autoprefixer({ browsers: ['> 1% in US'] });
+
+    return sass('sass/main.scss', { sourcemap: true })
+        .on('error', function (err) {
+            console.error('Sass error', err.message);
```

Comment:
I don't think this PR will have changed anything in regards to how we handle sass errors.

That said, I think it is definitely a good idea to exit with a non-zero error code if the build fails (for production builds, for debug builds I think we should just warn).  I can see if it is easy to do, if not I'll open a separate issue.